catalog/custom_validators.py

```diff
@@ -0,0 +1,29 @@
+import re
+
+from django.core.exceptions import ValidationError
+from django.core.validators import RegexValidator
+
+
+class Validator(RegexValidator):
+    FORBIDDEN_WORDS: list[str] # список запрещенных слов
+
+    FORBIDDEN_WORDS = ['казино',
+                       'биржа',
+                       'обман',
+                       'криптовалюта',
+                       'дешево',
+                       'полиция',
+                       'крипта',
+                       'бесплатно',
+                       'радар',
+                       ]
+    inverse_match = True
+
+    def __call__(self, value):
+        for word in self.FORBIDDEN_WORDS:
+            self.regex = re.compile(rf'({word})', re.IGNORECASE)
+            regex_matches = self.regex.search(str(value))
```

Comment:
К строке можно не приводить, ведь ты валидируешь строки
```suggestion
            regex_matches = self.regex.search(value)
```